_schema.graphql

```diff
@@ -9624,15 +9620,30 @@ type PopularArtists {
   artists: [Artist]
 }
 
+type Price {
```

Comment:
#minor. I wonder if it would be useful to use more generic name to describe monetary fields. For example, tax amount may feel a little different than a price.

Reply:
That's a very valid point. I'm open to suggestions.

Reply:
MonetaryAmount?